storage/cleanup.go

```diff
@@ -177,13 +200,28 @@ func (m *cleanupManager) cleanupFilesetFiles(t time.Time) error {
 		earliestToRetain := retention.FlushTimeStart(n.Options().RetentionOptions(), t)
 		shards := n.GetOwnedShards()
 		if n.Options().NeedsFilesetCleanup() {
-			multiErr = multiErr.Add(m.cleanupNamespaceFilesetFiles(earliestToRetain, shards))
+			multiErr = multiErr.Add(m.cleanupNamespaceDataFiles(earliestToRetain, shards))
 		}
 	}
 	return multiErr.FinalError()
 }
 
-func (m *cleanupManager) cleanupNamespaceFilesetFiles(earliestToRetain time.Time, shards []databaseShard) error {
+func (m *cleanupManager) cleanupSnapshotFiles() error {
+	multiErr := xerrors.NewMultiError()
+	namespaces, err := m.database.GetOwnedNamespaces()
+	if err != nil {
+		return err
+	}
+	for _, n := range namespaces {
+		shards := n.GetOwnedShards()
+		if n.Options().NeedsSnapshotCleanup() {
+			multiErr = multiErr.Add(m.cleanupNamespaceSnapshotFiles(shards))
+		}
+	}
+	return multiErr.FinalError()
+}
+
+func (m *cleanupManager) cleanupNamespaceDataFiles(earliestToRetain time.Time, shards []databaseShard) error {
```

Comment:
nit: rename to `cleanupShardDataFiles`?

Reply:
`cleanupShardDataFiles` sounds like it might just cleanup for a single shard, as opposed to all shards for a given namespace

Reply:
how about `cleanupDataFilesForShards`?

Reply:
still applicable?